articles/aks/azure-files-csi.md

```diff
@@ -346,7 +346,10 @@ mountOptions:
 
 ### Create NFS file share storage class
 
-Create a file named `nfs-sc.yaml` and copy the manifest below. For a list of supported `mountOptions`, see [NFS mount options][nfs-file-share-mount-options]
+Create a file named `nfs-sc.yaml` and copy the manifest below. For a list of supported `mountOptions`, see [NFS mount options][nfs-file-share-mount-options].
+
+> [!NOTE]
+> `vers`, `minorversion`, `sec` has been preset since [AKS v20240123](https://github.com/Azure/AKS/releases/tag/2024-01-23) and no longer can be configured. See [discussion](https://github.com/kubernetes-sigs/azurefile-csi-driver/issues/1692). 
```

Comment:
`vers`, `minorversion`, `sec` is set in the beginning when azure file driver supports nfs file share mount, so you could just simply say:

`vers`, `minorversion`, `sec` has been preset by the Azure File CSI driver

Reply:
Thanks, I have made relative changes.